app/views/taxons/sections/_news_and_communications.html.erb

```diff
@@ -1 +1,32 @@
-<%= render partial: 'document_list_with_grid', locals: { documents: section[:documents], see_more_link: section[:see_more_link] } %>
+
+<% featured_item = section[:documents].shift(section[:promoted_content_count]).first %>
```

Comment:
Do you need the `.first` here?

Reply:
I think so - `section[:documents].shift(section[:promoted_content_count])` returns the first item in an array of hashes, so we need to get the first item from that array.